adops/setting-up-prebid-video-in-dfp.md

```diff
@@ -52,16 +52,25 @@ Be sure to duplicate your line item and video creative for each Prebid price buc
    https://prebid.adnxs.com/pbc/v1/cache?uuid=%%PATTERN:hb_uuid%%
    ```
 
+   {: .alert.alert-warning :}
+
+   This creative URL is **required** in order to show video ads.  It points to
+   a server-side cache hosted by Prebid.org.  For more information
+   about why the cache is needed, see the note below:
+   [Prebid Cache and the VAST creative URL warning](#prebid-cache-note).
+
 4. Set the **duration** to **1**
 
 The resulting creative should look something like the following:
 
 {: .pb-img.pb-md-img :}
 ![DFP Video Creative Setup]({{site.baseurl}}/assets/images/ad-ops/dfp-creative-setup/dfp-creative-setup-04.png)
 
-{: .alert.alert-warning :}
-**VAST creative URL warning**  
-DFP will show you a warning that fetching VAST from the creative URL failed.  This is expected, since the creative URL above points to a cache hosted by Prebid Server.  Why do we do this caching?  For several reasons:  
+<a name="prebid-cache-note" />
+
+{: .alert.alert-info :}
+**Prebid Cache and the VAST creative URL warning**  
+DFP will show you a warning that fetching VAST from the creative URL failed.  This is expected, since the creative URL above points to a server-side cache hosted by Prebid.org.  Why do we do this caching?  For several reasons:  
 &bull; Sets [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS) headers appropriately on the DFP VAST URL so video players can access the content across multiple devices/platforms.  When these headers are absent, video will fail on certain devices/browsers.  

```

Comment:
This doesn't really feel like the right place to discuss the benefits of Prebid Cache.

Here's let's just say that the DFP Warning message is expected because the VAST tag URL points to a server-side asset cache hosted by Prebid Server.